modules/appeals_api/README.yml

```diff
@@ -300,23 +320,30 @@ components:
         date:
           type: string
           format: date
+          description: Date the event occurred
           example: '2016-05-30'
         details:
+          description: Further information about the event
           type: object
     Issue:
       type: object
+      description: Conditions that would qualify a Veteran for this benefit
       properties:
         active:
           type: boolean
+          example: true
+          description: Whether the Veteran making the appeal is currently affected by this issue
         description:
           type: string
           example: Service connection, tinnitus
+          description: Name of the issue and whether it was connected to a Veteran's service
```

Comment:
"Whether it was connected to a Veteran's service" is itself a kind of issue. Should just be "description of the issue."